xreds/dataset_provider.py

```diff
@@ -77,8 +77,7 @@ def get_dataset(self, dataset_id: str) -> xr.Dataset:
                 logger.info(f"Applying extension {ext_name} to dataset {dataset_id}")
             ds = extension().transform_dataset(ds=ds, config=ext_config)
 
-        # TODO: For now this cache is disabled for testing with redis
-        # self.datasets[cache_key] = {"dataset": ds, "date": datetime.datetime.now()}
+        self.datasets[cache_key] = {"dataset": ds, "date": datetime.datetime.now()}
```

Comment:
Im not sure if this should be configurable, for now this cache shares the same cache as the redis and filecache for reference datasets